submissions/andrewklmn/html-css-cinema-booking/style.css

```diff
@@ -0,0 +1,410 @@
+body {
+  padding: 0;
+  margin: 0;
+
+  background-color: white;
+
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+}
+
+h1 {
+  margin-top: 10px;
+  margin-bottom: 5px;
+
+  font-size: 16px;
+  color: #333;
+}
+
+header {
+  display: block;
+  position: fixed;
+  top: 0;
+
+  margin: 0;
+  padding: 0;
+
+  width: 100%;
+
+  background-color: #343434;
+}
+
+.logo {
+  width: 80px;
+  height: 57px;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  width: 100%;
+}
+
+nav > ol {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 10px;
+
+  padding: 0;
+}
+
+.menu > li {
+  list-style-type: none;
+  padding-right: 10px;
+}
+
+.menu a {
+  color: white;
+}
+
+.menu a:hover {
+  color: greenyellow;
+}
+
+main {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: stretch;
+
+  margin: 0;
+  margin-top: 61px;
+
+  width: 100%;
+  height: calc(100vh - 78px);
```

Comment:
Avoid using fixed height for blocks with dynamic content, Use `min-height` instead.